fpga/Makefile

```
@@ -58,21 +65,21 @@ $(BUILD_DIR)/build.ys $(BUILD_DIR)/twpm.json $(BUILD_DIR)/twpm_synth.v: $(SRC) $
 	@echo "synth_ecp5 -top $(TOP)" >> $(BUILD_DIR)/build.ys
 	@echo "write_verilog $(BUILD_DIR)/twpm_synth.v" >> $(BUILD_DIR)/build.ys
 	@echo "write_json $(BUILD_DIR)/twpm.json" >> $(BUILD_DIR)/build.ys
-	@yosys $(BUILD_DIR)/build.ys |& tee $(BUILD_DIR)/yosys.log
+	@yosys $(YOSYS_ARGS) $(BUILD_DIR)/build.ys |& tee $(BUILD_DIR)/yosys.log
 
 $(BUILD_DIR)/nextpnr.config: $(BUILD_DIR)/twpm.json $(LPF_FILE)
 	@nextpnr-ecp5 $(NEXTPNR_ARGS) |& tee $(BUILD_DIR)/nextpnr.log
 
 $(BUILD_DIR)/twpm.bit: $(BUILD_DIR)/nextpnr.config
 	@ecppack  --bootaddr 0   --compress $< --svf $(BUILD_DIR)/twpm.svf --bit $@
 else ifeq ($(TOOLCHAIN),diamond)
-$(BUILD_DIR)/build.tcl: $(BUILD_DIR) $(NEORV32_SRC) $(NEORV32_WRAPPER) $(BUILD_DIR)/bootloader.vhd
+$(BUILD_DIR)/build.tcl: $(BUILD_DIR) $(NEORV32_SRC) $(NEORV32_WRAPPER) $(NEORV32_BOOTLOADER)
 	@echo "prj_project new -name twpm -impl $(BOARD) -dev LFE5U-25F-8MG285C -synthesis synplify" > $@
 	@for file in $(SRC); do \
 		echo "prj_src add -work twpm $$(readlink -f $$file)" >> $@; done
 	@for file in $(NEORV32_SRC); do \
 		echo "prj_src add -work neorv32 $$(readlink -f $$file)" >> $@; done
-	@echo "prj_src add -work neorv32 $$(readlink -f $(BUILD_DIR)/bootloader.vhd)" >> $@
+	@echo "prj_src add -work neorv32 $$(readlink -f $(NEORV32_BOOTLOADER)" >> $@
```

Missing parenthesis, this broke build on diamond.
```suggestion
	@echo "prj_src add -work neorv32 $$(readlink -f $(NEORV32_BOOTLOADER))" >> $@
```